main.js

```diff
@@ -0,0 +1,157 @@
+// 1 //
+
+const validity = function (value) {
```

Comment:
Try to use the function naming convention

- use descriptive names, usually verbs in the imperative form.
- common prefixes are `get`, `make`, `apply` `validate` etc.
- common prefixes for functions, that return a boolean as a result are `is`, `has`